### ISSUE TITLE
Ajax Functionalities

### ISSUE THREAD

**Issue by UstymUkhman** [UstymUkhman/JSUtils#2]
Implement an easy way for XMLHttpRequests like jQuery.ajax():

http://www.w3schools.com/ajax/ajax_xmlhttprequest_create.asp
http://api.jquery.com/jquery.ajax/
https://github.com/jquery/jquery/blob/master/src/ajax.js
http://james.padolsey.com/jquery/#v=1.6.2&fn=jQuery.ajax

**Comment by TheJoin95** [comment]
Ehy oh, man!
I can do this as well.. One questions:
Do you want to implement all function/funtionality of jquery.ajax? Or just the way you can calls it?
I mean: callback, jsonp, abort, success.. i want to implement all of these function and parameters. Do you want it too?

**Comment by UstymUkhman** [comment]
Yes, exactly, all functions/funtionalities (or most of them, the most important and useful ones to start) of jQuery.ajax().
Parameters and callback functions should have same names as [these ones](http://api.jquery.com/jquery.ajax/) and the call is:

```javascript
var u = new Utils();
...
u.ajax({
    ...
});
```

Feel free to consult any of likns in the comment above (for example the [jQuery implementation of XMLHttpRequests](https://github.com/jquery/jquery/blob/master/src/ajax.js)), open other issues for this request or for any functionality you'd like to see implemented natively in JavaScript.
